apps/libexec/modules/nagios_command.py

```diff
@@ -957,6 +957,16 @@ class nagios_command:
 			'brief': 'You are trying to execute an unsupported command.',
 			'template': 'CHANGE_CONTACT_MODSATTR;contact_name;value',
 		},
+		'DEL_DOWNTIME_BY_HOST_NAME': {
+			'nagios_id': 170,
+			'description': 'This command is used to delete a service or a host downtime.\n'
+            '"service_description", "downtime_start_time" and "comment" can be set to an empty string to match all downtimes.\n'
```

Comment:
spaces as indents here, vs tabs the rest of the file.

Reply:
Fixed